vulnfeeds/cmd/combine-to-osv/main.go

```diff
@@ -42,13 +48,46 @@ func main() {
 	}
 
 	allCves := loadAllCVEs(*cvePath)
-	allParts := loadParts(*partsInputPath)
-	combinedData := combineIntoOSV(allCves, allParts, *cveListPath)
+	allParts, cveModifiedMap := loadParts(*partsInputPath, *inputBucket)
+	combinedData := combineIntoOSV(allCves, allParts, *cveListPath, cveModifiedMap)
 	writeOSVFile(combinedData, *osvOutputPath)
 }
 
+// getModifiedTime gets the modification time of a given GCS file
+func getModifiedTime(innerPath string, bucket string) (time.Time, error) {
```

Comment:
Do we need to query GCS for the modified time? I think there's a way to copy the modified time when downloading the records (it might already be the case / is default behaviour), and we can just read it straight from disk.

Reply:
Ah, good point! I will update this